### PR TITLE
CI: make sure all libraries that use exceptions declare so.

### PR DESCRIPTION
**Pull request by hzeller**
Signed-off-by: Henner Zeller <h.zeller@acm.org>

### CODE REVIEW

**Comment by hzeller** [comment]
So essentially, this pull request is a no-op as it just ends up adding a comment. Once we have reduced the Kythe dependencies, we can uncomment that line. For now it is essentially just a documentation.

File: .github/bin/script.sh [hunk 1]
```diff
@@ -18,6 +18,13 @@ set -e
 
 source ./.github/settings.sh
 
+# Make sure we don't have cc_library rules that use exceptions but do not
```
Comment (mithro) [inline .github/bin/script.sh:21]: This should reference a github issue which points to these lines to be uncommented when the issue is resolved.

Comment (hzeller) [reply .github/bin/script.sh:21]: Done.

File: .github/bin/script.sh [hunk 2]
```diff
@@ -18,6 +18,14 @@ set -e
...
+# declare copts = ["-fexceptions"] in the rule. We want to make it as simple
+# as possible to compile without exceptions.
+# ... and unfortunately we have to disable this for now as the external
+# kythe dependency does use exceptions.
+# TODO(hzeller): Uncomment after  #747 is flying (excuse the pun)
```
Comment (mithro) [inline .github/bin/script.sh:26]:
```suggestion
# TODO(hzeller): Uncomment after https://github.com/google/verible/issues/747 is fixed.
```

**Pull request merged**